clas/README.md

```diff
@@ -0,0 +1,19 @@
+# Contributor License Agreements (CLAs)
+
+To help protect the project owners and contributors to the project, we've established a common set of CLAs.
+The Individual Contributor Agreement should be signed by anyone submitting code to the project.
+The Corporate Contributor Agreement should be signed by any organization who's engineers are submitting code on their behalf.
+
+### Individual Contributors
+
+For the individual contributor agreement, we leverage [cla-assistant.io](https://cla-assistant.io).
+A copy of this agreement can be found in [individual.md](individual.md).
+This agreement will need to be signed by anyone looking to contribute code to the project.  
+To sign this agreement, you can go here and sign using your github account:
+
+https://cla-assistant.io/deps-cloud/deps.cloud
+
+### Corporate Contributors
+
+For the corporate contributor agreement, please complete the [corporate.md](corporate.md) and email it to us at [clas@deps.cloud](mailto:clas@deps.cloud).
```

Comment:
Looking at the corporate CLA, it's not clear what I would do to sign on behalf of my employer. I'd recommend adding lines to collect the name of the signer, their title, the date they signed, and their work email.